lib/Sema/TypeCheckProtocol.cpp

```diff
@@ -3444,10 +3444,12 @@ ConformanceChecker::checkActorIsolation(ValueDecl *requirement,
 
   // Complain that this witness cannot conform to the requirement due to
   // actor isolation.
-  witness->diagnose(diag::actor_isolated_witness,
-                    isDistributed && !isDistributedDecl(witness),
-                    refResult.isolation, witness, requirementIsolation)
-    .limitBehaviorUntilSwiftVersion(behavior, 6);
+  witness
+      ->diagnose(diag::actor_isolated_witness,
+                 isDistributed && !isDistributedDecl(witness),
+                 refResult.isolation, witness, requirementIsolation,
+                 Conformance->getProtocol())
+      .limitBehaviorUntilSwiftVersion(behavior, 6);
```

Comment:
Should we also add a note to point at the requirement?

Reply:
Ah, we already emit one:

https://github.com/swiftlang/swift/blob/b718c6d448104d7365ebf1c48eb5f0605631ab19/lib/Sema/TypeCheckProtocol.cpp#L3535

But "x declared here" is not as helpful and clear as I would like it to be, especially when there are other notes attached. I will swap it for a new note that says "requirement x declared here".

Reply:
Sounds good! Could be done separately as well.